src/NHibernate.Test/TypeParameters/TypeParameterTest.cs

```diff
@@ -93,17 +94,18 @@ public void Loading()
 
 			Widget obj = (Widget) s.CreateQuery("from Widget o where o.Str = :string")
 			                      	.SetString("string", "all-normal").UniqueResult();
-			Assert.AreEqual(obj.ValueOne, 7, "Non-Default value incorrectly loaded");
-			Assert.AreEqual(obj.ValueTwo, 8, "Non-Default value incorrectly loaded");
-			Assert.AreEqual(obj.ValueThree, 9, "Non-Default value incorrectly loaded");
-			Assert.AreEqual(obj.ValueFour, 10, "Non-Default value incorrectly loaded");
+			Assert.AreEqual(7, obj.ValueOne, "Non-Default value incorrectly loaded");
+			Assert.AreEqual(8, obj.ValueTwo, "Non-Default value incorrectly loaded");
+			Assert.AreEqual(9, obj.ValueThree, "Non-Default value incorrectly loaded");
+
```

Comment:
```suggestion
```

Reply:
I'll redo the commits so it is done by the tool rather than with (occasional) manual fixes.